changelogs/fragments/2684-open_iscsi-single-target-multiple-portal-overrides.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - open_iscsi - add login in check for specific portals (https://github.com/ansible-collections/community.general/issues/2683).
```

Could you elaborate on what check this PR has added?
Feel free to change my suggestion.
```suggestion
  - open_iscsi - also consider ``portal`` and ``port`` to check if already logged in or not (https://github.com/ansible-collections/community.general/issues/2683).
```